test/Tester/Tester.csproj

```diff
@@ -30,4 +30,8 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
+
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
```

Comment:
shouldn't be included, not sure why your environment is adding these

Reply:
good catch.